app/src/main/java/com/sopt/now/compose/presentation/Login/LoginViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.sopt.now.compose.presentation.Login
+
+import android.content.ContentValues.TAG
+import android.util.Log
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.compose.domain.AuthRepository
+import com.sopt.now.compose.domain.model.AuthEntity
+import kotlinx.coroutines.launch
+import retrofit2.HttpException
+
+class LoginViewModel(
+    private val authRepository: AuthRepository,
+) : ViewModel() {
+
+    val loginResult: MutableLiveData<Boolean> = MutableLiveData()
```

Comment:
```suggestion
    private val _loginResult = MutableLiveData<Boolean>()
    val loginResult: LiveData<Boolean> get() = _loginResult
```
이렇게 작성해서 _loginResult는 ViewModel에서만 수정 가능하게 하고 activity에서는 loginResult에 접근만 가능할 수 있게 하면 좋을 것 같아요!

Reply:
errorMessage도 마찬가지 에요!!